refinery/ui/bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "refinery-platform-ui",
-  "version": "1.1.0",
```

Comment:
What's the reason?

Reply:
This change isn't needed, you're right. I'll fix now